spec.bs

```diff
@@ -663,6 +664,11 @@ dictionary AuctionAdConfig {
 };
 </xmp>
 
+Note: To help with ease of adoption, until TBD we will support the attribute
+{{Window/navigator}}.{{Navigator/deprecatedRunAdAuctionEnforcesKAnonymity}},
+which is true when {{Window/navigator}}.{{Navigator/runAdAuction()}} enforces
+k-anonymity when running an auction and false otherwise.

```

Comment:
nit: link "k-anonymity" to the k-anon section of the spec.
[[#k-anonymity]]

Reply:
Done.